recipes-wam/wam/wam-tinyproxy/tinyproxy.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
Why do we need this? We should not run any tinyproxy system service.

Reply:
We are not using this file and it was dead code. So remove this file and amended commit.